cli/src/lib.rs

```diff
@@ -375,6 +375,9 @@ pub enum IdlCommand {
     },
     Close {
         program_id: Pubkey,
+        /// The IDL account to close. If none is given, then the IDL account derived from program_id is used.
+        #[clap(short, long)]
```

Comment:
It would be better to only have the long form because `-i` is too unclear for an IDL command.

```suggestion
        #[clap(long)]
```